tensilelite/Tensile/TensileInstructions/Instructions.py

```diff
@@ -1813,11 +1832,19 @@ def setupInstructions(self):
             lgkmcnt = 0
             vmcnt   = 0
             vscnt   = 0
+            dscnt   = 0
+            kmcnt   = 0
+            loadcnt = 0
+            storecnt= 0
             comment = "(Wait all)"
         else:
             lgkmcnt = self.lgkmcnt
             vmcnt   = self.vmcnt
             vscnt   = self.vscnt
+            kmcnt   = self.kmcnt
```

Comment:
can it be self.lgkmcnt when self.kmcnt = -1?
If so, you don't need to change a lot of  "SWaitCnt(lgkmcnt=0)" as "SWaitCnt(lgkmcnt=0, kmcnt=0)".

Reply:
due to lgkmcnt split into dscnt and kmcnt.
I need one of kmcnt and dscnt info to separate lgkmcnt.
the times to use SWaitCnt for constant is less than lds r/w.